README.adoc

```diff
@@ -228,7 +228,7 @@ sudo apt install ovmf
 5. Run oe-selftest:
 +
 ```
-oe-selftest --run-tests updater
+oe-selftest -r updater_native -r updater_qemux86_64 -r updater_minnowboard -r updater_raspberrypi
```

Comment:
I don't think you have to specify `-r` each time.

Reply:
ok I'll try it